.travis.yml

```diff
@@ -5,10 +5,15 @@ python:
   - "3.5"
 install:
   - pip install --upgrade pip
+  - pip install pytest-cov
+  - pip install codecov
   - make develop
 script:
   - make quality
   - make test
+  - py.test --cov=./
```

Comment:
`make test` [already executes](https://github.com/edx/pa11ycrawler/blob/master/Makefile#L27) `py.test` 
I *think* you should be able to add the `--cov=./` there instead.